### PR TITLE
Merge in 8.2.7

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: ports/atmel-samd/boards/cp32-m4/mpconfigboard.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #define MICROPY_HW_BOARD_NAME "CP32-M4"
-#define MICROPY_HW_MCU_NAME "samd51j20"
+#define MICROPY_HW_MCU_NAME "samd51j20A"
```
Comment (dhalbert) [inline ports/atmel-samd/boards/cp32-m4/mpconfigboard.h:2]: Don't know why the 'A' was added , in uppercase? (I was off the livestream after a while.)

**Review by dhalbert — CHANGES_REQUESTED** [review]
Only one q

File: ports/atmel-samd/boards/cp32-m4/mpconfigboard.h (hunk 1 above)
Comment (tannewt) [reply ports/atmel-samd/boards/cp32-m4/mpconfigboard.h:2]: It is from: https://github.com/adafruit/circuitpython/pull/8414/files#diff-a7668631002dd4a35f2f0e0802af304e1dda1754d0fd114d4f1996ddabe15150L2

I'll remove it.

**Review by dhalbert — APPROVED** [review]
Thanks!

**Pull request merged**